synapse/federation/sender/__init__.py

```diff
@@ -422,47 +404,38 @@ async def get_federatable_events_and_destinations(
 
     async def _distribute_events(
         self,
-        per_room_events_and_dests: List[
-            Tuple[str, List[Tuple[EventBase, Collection[str]]]]
-        ],
+        events_and_dests: List[Tuple[EventBase, Collection[str]]],
     ) -> None:
+        """Distribute events from the transmission loop.
+
+        Args:
+            events_and_dests: A list of tuples, which are (event, destinations).
```

Comment:
what are the destinations?

Reply:
Will add some more comments, and add types in that tuple